stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -112,8 +112,14 @@ internal class PaymentOptionsViewModel(
                         )
                     }
                 },
-                onFailure = {
-                    // TODO(michelleb-stripe): Handle failure cases
+                onFailure = { throwable ->
+                    // TODO(michelleb-stripe): Failed to create payment method or attach it to the payment should we report it as success anyway?
+
```

Comment:
I am not sure how to force this error on iphone.  I might need to wait until tomorrow to ask if I don't have luck sifting through the source code.